examples/gcp_defaults/main.tf

```diff
@@ -0,0 +1,26 @@
+provider "vault" {
+  address = "http://localhost:8200"
+  token   = "root"
+}
+
+locals {
+  project = "test"
```

Comment:
If the local value is not intended to change, use a `variable`

Reply:
fixed